### DISCUSSION TITLE
Manually add browsers to the browser's list

### DISCUSSION THREAD

**Discussion by psudo-dev** [will-stone/browserosaurus#697] [Q&A]
I made some browsers with Coherence X but they are not appearing in the list, but they are the main browsers I use, is there a way to manually add them? Thanks.

**Comment by bigornoo** [comment]
+1 for Zen browser (based on Firefox).

**Reply by psudo-dev** [reply]
I ended up switching to Velja

**Reply by Lockszmith-GH** [reply]
$8.00 for a non-open-source project. Feels all kind of wrong. But I completely understand.
Wish these FOSS projects get this type of support instead.

**Reply by psudo-dev** [reply]
When I got it Velja was still free though.

**Reply by Lockszmith-GH** [reply]
Nice, enjoy!
